e2e/test/scenarios/visualizations-charts/reproductions/43075-dropdown-overflows.cy.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import {
+  createQuestion,
+  popover,
+  restore,
+  type StructuredQuestionDetails,
+} from "e2e/support/helpers";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const questionDetails: StructuredQuestionDetails = {
+  name: "20548",
```

Comment:
```suggestion
  name: "43075",
```

Or we can skip the name altogether

```suggestion
```